package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "angular-ngrx-chat-api",
+  "version": "1.0.0",
+  "description": "Chat API for Angular NGRX lecture",
+  "main": "server.js",
+  "scripts": {
+    "clean": "rm -rf build",
+    "build": "npm run clean && npx tsc",
+    "dev": "nodemon",
+    "docker:start": "docker compose up",
+    "docker:start:dev": "docker compose up -d && docker exec -it angular-ngrx-chat-api sh",
+    "docker:stop": "docker compose down",
```

Comment:
I usually add `-v` to the command so that shared volumes are removed on shutdown. I've run into issues with data within a volume being stale after a restart.